WordPress/Classes/Services/PostService.m

```diff
@@ -593,6 +389,82 @@ - (AbstractPost *)findPostWithID:(NSNumber *)postID inBlog:(Blog *)blog {
     return [posts firstObject];
 }
 
+- (NSDictionary *)remoteSyncParametersDictionaryForRemote:(nonnull id <PostServiceRemote>)remote
```

Comment:
> Most of the parameters are the same, and only a couple aren't supported at all within XML-RPC.

True, but that's not a compelling reason to more tightly couple the service to the particular needs of the remotes. 

> Perhaps if XML-RPC supported everything REST did we would just have a key/value translator for each remote

I think we could do this without all keys supported for each remote. 

> Otherwise, PostService doesn't necessarily know that XML-RPC can't handle a search and should handle it appropriately

I'm not seeing where its doing anything special currently.  What am I missing?  The remotes already manage implementation differences in RemotePosts (e.g. `parametersWithRemotePost:`). Seems like they should handle differences in these parameters as well.